test/platform.test.ts

```diff
@@ -20,4 +22,223 @@ suite('Platform Tests', () => {
             done();
         });
     });
+
```

Comment:
I mean I have no problem with it, but it means we are mixing assertion libraries.